TestApps/PlacesTestApp/ViewController.swift

```diff
@@ -62,6 +62,10 @@ class ViewController: UIViewController {
         }
     }
     
+    @IBAction func setAccuracyAuthorization(_ sender: Any) {
+        Places.setAccuracyAuthorization(accuracy: .fullAccuracy)
```

Comment:
You might need to add as this API is only available 14+
```
        if #available(iOS 14, *) {
            Places.setAccuracyAuthorization(accuracy: .fullAccuracy)
        }
```

same for the ObjectiveC sample app